### PR TITLE
Add vampires.rb file

### PR DESCRIPTION
**Pull request by powerwlsl**
Hi, Tyler

I've just finished 4.4 solo challenge :-)  I feel like I'm a vampire...

Can you review my code :)?

Thanks! 

@bontgoy

### CODE REVIEW

File: ruby/vampires.rb [hunk 1]
```diff
@@ -1,15 +1,76 @@
...
+				Should we order some for you? (yes/no)"
+		garlic = gets.chomp
+
+		puts "Would you like to enroll in the company’s 
+				health insurance?(yes/no)"
+		insurance = gets.chomp
+
+
+		name_downcase = name.downcase
+		age_boolean = (age.to_i == (2016-year.to_i))
```
Comment (Bontgoy) [inline ruby/vampires.rb]: There is a more dynamic way to work with the year.  Google the ruby `Time` class.  You can use commands like `Time.now.year` to make this code scalable in such a way that you would not need to rewrite this file every new year

File: ruby/vampires.rb [hunk 2]
```diff
@@ -1,15 +1,76 @@
...
+		
+	end
+
+	# Detection logic
+
+		if (name_downcase.include? "cula") || (name_downcase.include? "fang")
+
+			puts "Definitely a vampire."
+
+		elsif age.to_i != (2016-year.to_i) && garlic == "no" && insurance == "no"
```
Comment (Bontgoy) [inline ruby/vampires.rb]: Same with `Time` class

File: ruby/vampires.rb [hunk 3]
```diff
@@ -1,15 +1,76 @@
...
+			
+			puts "Almost certainly a vampire."
+
+		elsif age.to_i != (2016-year.to_i) && garlic == "no"
```
Comment (Bontgoy) [inline ruby/vampires.rb]: ^^

File: ruby/vampires.rb [hunk 4]
```diff
@@ -1,15 +1,76 @@
...
+
+			puts "Probably a vampire"
+
+		elsif age.to_i == (2016-year.to_i) && garlic == "yes"
```
Comment (Bontgoy) [inline ruby/vampires.rb]: ^^

**Comment by Bontgoy** [comment]
@powerwlsl excellent work with control flow.  Check out the Time class suggestions I made.  Reach out if you have any questions.

File: ruby/vampires.rb (hunk 4 above)
Comment (powerwlsl) [inline ruby/vampires.rb]: Thank you for your comments Tyler :-)

**Pull request merged**